src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -193,7 +203,9 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
                   </div>
                 )}
               </div>
-              {seatedAtTable && (
+
+              {/* @debt This should probably be all rolled up into a single canonical component for emoji reactions/etc*/}
+              {seatedAtTable && shouldShowReactions && (
```

Comment:
My previous review comment said to extract _both_ of these into a _single_ `const`..

```suggestion
              {shouldShowReactions && (
```